msg/LandingTargetPose.msg

```diff
@@ -24,3 +24,13 @@ bool abs_pos_valid		# Flag showing whether absolute position is valid
 float32 x_abs			# X/north position of target, relative to origin (navigation frame) [meters]
 float32 y_abs			# Y/east position of target, relative to origin (navigation frame) [meters]
 float32 z_abs			# Z/down position of target, relative to origin (navigation frame) [meters]
+
+bool abs_vel_valid		# Flag showing whether absolute velocity is valid
+float32 vx_abs			# X/north velocity of target, relative to origin (navigation frame) [meters/second]
```

Comment:
The velocity is probably not "relative to the origin", but rather "in the navigation frame of the target". Or just don't specify it, it's enough to tell "X/north velocity of target"